ga4gh/cli/repomanager.py

```diff
@@ -1,6 +1,8 @@
 """
 repo manager cli
 """
+
+
```

Comment:
Don't put spaces like this between the top comment and the import lines.  It's unnecessary and it creates a diff for files that you didn't really make any changes in, making the PR harder to review.  Please fix that for all other files below as well when you resubmit.